samples/matter/light_switch/Kconfig.sysbuild

```diff
@@ -7,4 +7,7 @@
 source "${ZEPHYR_BASE}/share/sysbuild/Kconfig"
 
 config NRF_DEFAULT_MULTIPROTOCOL
-	default y
+	default y if ($(BOARD) = "nrf5340dk_nrf5340_cpuapp")
```

Comment:
Question: if we build the nrf5340 with nrf7002ek we should expect to have multiprotocol rpmsg?

Reply:
Good point, initially I though the nrf7002ek was out of scope, but now I guess I can be handled. I will open the next PR for that. The same will apply to the Matter weather station and nrf7002eb, but this needs to wait until we have OTA support for sysbuild builds, because applications do not support non-DFU configuration.

Reply:
Ok, I fixed it here.